content/en/security/application_security/troubleshooting.md

```diff
@@ -369,6 +369,9 @@ Enable debug logs with the environment variable `DD_TRACE_DEBUG=1`. The ASM libr
 
 **Note:** ASM only outputs logs when it is enabled. Use the environment variable `DD_APPSEC_ENABLED=1` to enable ASM.
 
+#### Crashes in go-libddwaf code
+
+If you see segfaults happening during `github.com/DataDog/go-libddwaf` module execution, please consider enabling CGO using the environment variable `CGO_ENABLED=1` during compilation. If the issue persist, please consider opening an issue..
```

Comment:
This is weird to read imo: if I observe crashes on my app due to appsec, I won't try anything more until it gets fixed for my setup, no? Why would I trust it again with CGO_ENABLED=1?

Reply:
Yeah I don't know, I would try some stuff personally. Want to write a suggestion or just to remove this section ?

Reply:
I think a crash is a crash and should be fixed, not worked around like that.
So yes I think we should remove this section, as it otherwise means we kind of expect this behaviour.